packages/workbox-cache-expiration/src/lib/cache-expiration.js

```diff
@@ -159,15 +162,31 @@ class CacheExpiration {
 
         const parsedDate = new Date(dateHeader);
         // If the Date header was invalid for some reason, parsedDate.getTime()
-        // will return NaN, and the comparison will always be false. That means
-        // that an invalid date will be treated as if the response is fresh.
-        if ((parsedDate.getTime() + (this.maxAgeSeconds * 1000)) < now) {
-          // Only return false if all the conditions are met.
-          return false;
-        }
+        // will return NaN, and the comparison will always be false. We want
```

Comment:
I found opting for the negation strategy here a little hard to parse. I'm similarly having trouble reasoning why we're not just special casing NaN per Matt's comment.